docs/manual.md

```diff
@@ -138,7 +145,7 @@ automatically:
    upper case letters (`Assets:test` becomes `Assets:Test`)
 3. Strips accents and umlauts because they are currently not
    supported in beancount ([issue
-   171](https://bitbucket.org/blais/beancount/issues/171)).
+   171](https://bitbucket.org/blais/beancount/issues/161)).
```

Comment:
This looks wrong, either it's 161 or 171… but it should be the same in anchor and link ;)

Reply:
Thanks!  Glad you're reviewing my changes.